tests/test_pattern_rewriter.py

```diff
@@ -435,6 +432,7 @@ def test_operation_deletion_reversed():
     expected = \
 """builtin.module() {}"""
 
+    @op_type_rewrite_pattern
```

Comment:
this is the only breaking change as of the current state of the PR: the functions that are passed in to the AnonymousRewritePattern need to have three arguments. The client needs to either have the first argument be the pattern instance or annotate the function with the decorator